Add a no-browser mode to gh browse

silby opened the pull request:
For when you just want the destination URL on stdout.

That's it!

silby committed:
Add a no-browser mode to gh browse

For when you just want the destination URL on stdout.

cliAutomation commented:
Hi! Thanks for the pull request. Please ensure that this change is linked to an issue by mentioning an issue number in the description of the pull request. If this pull request would close the issue, please put the word 'Fixes' before the issue number somewhere in the pull request body. If this is a tiny change like fixing a typo, feel free to ignore this message.

Commit:
don't check Fprintf error

we don't ever check the return of Fprintf anywhere else in the codebase
so doing it here suggests that it's a special case. if it's something we
should be doing we can circle back and do it more consistently.

vilmibm reviewed (approved):
nice! thank you~
